### PR TITLE
[GH-244] Update autocomplete help summary

### PR DESCRIPTION
**Pull request by jeromegrosse**
#### Summary
Update the auto-complete commands. In particular, states that the `meeting topic` argument is optional. The update is performed both in auto-complete themselves as well as the text returned by `help`.

| Autocomplete | Help Output |
|---------------|-------------|
| image | image |

In addition, the general autocomplete description reflects the plugin and user capabilities to connect/disconnect:
| Can Connect | Cannot Connect |
|--------------|------------------|
| image | image |

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-zoom/issues/244

### CODE REVIEW

File: server/command.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ func (p *Plugin) getCommand() (*model.Command, error) {
 	return &model.Command{
 		Trigger:              "zoom",
 		AutoComplete:         true,
-		AutoCompleteDesc:     "Available commands: start, disconnect, help",
+		AutoCompleteDesc:     "Available commands: start, connect, disconnect, help",
```
Comment (larkox) [inline server/command.go:38]: Connect and disconnect are available only on certain configurations. @hanzei do you think we can fill this conditionally depending on the configuration? Should we leave that to a different PR?

Comment (jeromegrosse) [reply server/command.go:38]: Do you think it would be okay to reuse the same condition (`p.configuration.EnableOAuth && !p.configuration.AccountLevelApp`) that we have in `getAutocompleteData`? I can do it in the same PR if you think it's ok!

Comment (hanzei) [reply server/command.go:38]: I hope @mickmister will cover that in https://github.com/mattermost/mattermost-plugin-zoom/pull/249. This is how GitLab handles that case: https://github.com/mattermost/mattermost-plugin-gitlab/blob/e979912e580919b8d0b6090d1f5c02054dd53c86/server/command.go#L612-L619

File: server/command.go [hunk 2]
```diff
@@ -237,13 +237,13 @@ func (p *Plugin) getAutocompleteData() *model.AutocompleteData {
...
 
-	start := model.NewAutocompleteData("start", "[meeting topic]", "Starts a Zoom meeting")
+	start := model.NewAutocompleteData("start", "[meeting topic]", "Starts a Zoom meeting with a topic (optional)")
 	zoom.AddCommand(start)
 
 	// no point in showing the 'disconnect' option if OAuth is not enabled
 	if p.configuration.EnableOAuth && !p.configuration.AccountLevelApp {
 		connect := model.NewAutocompleteData("connect", "", "Connect to Zoom")
-		disconnect := model.NewAutocompleteData("disconnect", "", "Disonnects from Zoom")
+		disconnect := model.NewAutocompleteData("disconnect", "", "Disonnect from Zoom")
```
Comment (justinegeffen) [inline server/command.go:246]:
```suggestion
		disconnect := model.NewAutocompleteData("disconnect", "", "Disconnect from Zoom")
```

File: server/command.go (hunk 1 above)
Comment (mickmister) [reply server/command.go:38]: @hanzei This is a bit different as this isn't an "incomplete" setup scenario, it's really an "alternative" setup scenario.

The `connect` command may be purposely unavailable even though the plugin is in a fully-configured state. 2/5 I'm thinking this conditional check for showing the `connect` command should go in this PR.

Comment (hanzei) [reply server/command.go:38]: I'm fin with handling the `connect` case in this PR. :+1:

**Review by hanzei — COMMENTED** [review]
@jeromegrosse Would you be open doing the change proposed in https://github.com/mattermost/mattermost-plugin-zoom/pull/251/files#r803570554 as part of this PR?

**Comment by jeromegrosse** [comment]
@hanzei, sure! I'll do my best to update the PR today.

**Comment by jeromegrosse** [comment]
So, there's actually already a method in that very file that checks for the condition: `func (p *Plugin) canConnect(user *model.User) bool`.
This method depends on a User object pointer. I've thought of using it with `canConnect(nil)` and check within that method if `user != nil`. However, I don't find it super explicit. 
What do you all think?

**Review by mickmister — APPROVED** [review]
@jeromegrosse For the `/zoom help` text, we can check if the user is able to connect. But the autocomplete suggestions not able to be user-secific, so we can only look at the configuration in the autocomplete case. The autocomplete suggestions are arguably more visible than the help text, because the user has to go out of their way to show the help text.

I think we should keep these two consistent, so your most recent commits make sense to me. Approving, thanks @jeromegrosse

**Comment by dipak-demansol** [comment]
@jeromegrosse As Per The Description When i'm Disconnected then i'm Expecting this 
image
                           
But i'm still getting command like this 
image

**Comment by mickmister** [comment]
@dipak-demansol The plugin's command autocomplete API is not able to change those descriptions per-user, so it will need to stay this way

**Comment by hanzei** [comment]
@jeromegrosse Sorry for the delay on your PR. Would you be open on merging `master` into your branch? Then your PR can get merged.

**Comment by jeromegrosse** [comment]
Sorry for the late reply. Let me set again the plugin on my machine and I'll solve the conflicts.

**Comment by hanzei** [comment]
Aweseome, thank you very much for the contribution @jeromegrosse
